pocketfft_hdronly.h

```diff
@@ -402,20 +402,18 @@ struct util // hack to avoid duplicate symbols
     return result*double(ni);
     }
 
-  /* returns the smallest composite of 2, 3, 5, 7 and 11 which is >= n */
+  /* inner workings of good_size_cmplx() */
   template<typename UIntT>
-  static POCKETFFT_NOINLINE UIntT good_size_cmplx(UIntT n)
+  static POCKETFFT_NOINLINE UIntT good_size_cmplx_typed(UIntT n)
     {
-    static_assert(std::numeric_limits<UIntT>::is_integer && (!std::numeric_limits<UIntT>::is_signed),
-      "type must be unsigned integer");
```

Comment:
I actually liked these asserts! Again, not a big deal, but a nice little layer of protection against a user calling these templated functions directly.

Reply:
It certainly can't hurt to leave them in ... I just assumed that these functions would be so obscure that no one would dare call them from outside :-)

Reply:
That's the intention, but I've learned to not underestimate the "resourcefulness" of people. 🤣